Source/Operations/AddDefaultConstraintOperation.cs

```diff
@@ -22,7 +22,8 @@ public sealed class AddDefaultConstraintOperation : ConstraintOperation
 
 		public override string ToIdempotentQuery()
 		{
-			return $"if object_id('{SchemaName}.{Name}', 'D') is null{Environment.NewLine}\t{ToQuery()}";
+			return $"if object_id('{SchemaName}.{Name}', 'D') is null{Environment.NewLine}" +

```

Comment:
Other similar constructs use BEGIN/END; adding such might increase uniformity.

Reply:
I only use BEGIN/END in situations where the query that executes when the conditional is true spans multiple lines.